### ISSUE TITLE
VARIO calculation broken

### ISSUE THREAD

**Issue by pventafridda** [LK8000/LK8000#854]
The instant vario is totally broken. Compared to V5, replaying the same IGC, on v6 we see values jumping 28 m/s.
Update: cannot replicate it in second test.

**Comment by pventafridda** [comment]
in V6 the VARIO IS NOT CORRECT. 
All calculations using the vario are thus incorrect too.
(netto vario, MC etc.)

**Comment by brunotl** [comment]
interesting, last change in Vario Calculation : 11/12/2012

**Comment by brunotl** [comment]
what is your test case ?

**Comment by pventafridda** [comment]
My test case is v5.0a.
We did not have any bug open on this, no problems reported, and consolidated working since 7 years. 
In 7 years we did hundreds of hours of tests. Several thousands of pilots have flown with it, and I have made over 800 hours in flight with absolutely no problems.
So if now the vario is not showing the same, while replaying an IGC file, then I assume it is not necessarily correct.
I hope to be wrong!
